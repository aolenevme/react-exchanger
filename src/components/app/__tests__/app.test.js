import React from "react";
import ShallowRenderer from "react-test-renderer/shallow.js";

import App from "../app.js";

describe("<App />", () => {
    it("is rendered correctly", () => {
        const renderer = new ShallowRenderer();

        renderer.render(<App />);

        expect(renderer.getRenderOutput()).toMatchSnapshot();
    });
});
