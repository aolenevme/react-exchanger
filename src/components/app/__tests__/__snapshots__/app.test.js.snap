// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> is rendered correctly 1`] = `
<app__Wrapper>
  <app__Header>
    <Button>
      Cancel
    </Button>
    <Memo(RateSelector) />
    <Button>
      Exchange
    </Button>
  </app__Header>
  <Memo(CarouselFactory) />
  <app__Triangle />
  <app__TargetCarouselFactory
    areTargetPockets={true}
  />
</app__Wrapper>
`;
